Update Wild_Spirits.md

Rioskill opened the pull request:
Протестировал создание и редактирование мероприятия

Rioskill committed:
Update Wild_Spirits.md

Протестировал создание и редактирование мероприятия

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -84,6 +84,18 @@
+- Сайт
+  - Если сайт имеет неверный формат, то после сохранения
+    - отображается как до изменения
+    - никаких подсказок не появляется
```
Такие штуки как баги не надо помечать? Не так плохо, как некоторые моменты, конечно

Rioskill replied:
Пометил

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,66 @@
+- При нажатии на кнопку "создать мероприятие" пользователем, не являющимся организатором, открывается модальное окно организатора
+- При нажатии на кнопку "стать организатором" без заполнения поля "Телефон" появляется подсказка "телефон не указан"
+- При заполнении поля "Телефон" невалидными данными (наличие букв, спец. символов) появляется подсказка "Неправильный номер телефона"
+- ${\color{red} При \ заполнении \ поля \ "Телефон" \ номером \ из \ 12 \ цифр, \ подсказдка \ не \ появляется, \ но \ в \ профиле \ отображаются \ только \ первые \ 11 \ символов }$
```
Опечатка
```suggestion
- ${\color{red} При \ заполнении \ поля \ "Телефон" \ номером \ из \ 12 \ цифр, \ подсказка \ не \ появляется, \ но \ в \ профиле \ отображаются \ только \ первые \ 11 \ символов }$
```

Rioskill replied:
Исправил

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,66 @@
+  - Появляется подсказка "Время начала обязательно"
+- При попытке создать мероприятие с временем начала позже времени окончания
+  - Мероприятие не создаётся
+  - Появляется посказка "Время окончания мероприятия не может быть раньше времени начала"
```
Опечатка, дальше еще раза 3-4 в том же слове подобное
```suggestion
  - Появляется подсказка "Время окончания мероприятия не может быть раньше времени начала"
```

Rioskill replied:
Спасибо, исправил

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,66 @@
+
+## Создание мероприятия
+- При попытке создать мероприятие без названия
+  - Мероприятие не создаётся
```
Стоит еще расписать положительный кейс - что нужно для успешного создания мероприятия и  что происходит после: появляется ли оно в ленте на главной, в профиле

Rioskill replied:
Расписал

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,66 @@
+- При попытке создать мероприятие с временем начала позже времени окончания
+  - Мероприятие не создаётся
+  - Появляется посказка "Время окончания мероприятия не может быть раньше времени начала"
+- Создание места
```
Аналогично стоит расписать положительный кейс. Написать, появляется ли место в поиске после успешного создания. Ещё я бы отметил, как баг, что место автоматически не выбирается после создания (приходится его искать)

Rioskill replied:
Оке, расписал

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,66 @@
+- При попытке создать мероприятие с временем начала позже времени окончания
+  - Мероприятие не создаётся
+  - Появляется посказка "Время окончания мероприятия не может быть раньше времени начала"
+- Создание места
```
Что будет, если попытаться создать место с уже существущим именем?

Rioskill replied:
Добавил кей

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,66 @@
+    - При наведении на изображение значок становится серым
+    - При нажатии на изображение открывается меню выбора фото
+
+## Редактирование мероприятия
```
Корректно ли работает изменение места проведения при редактировании?

Rioskill replied:
Добавил кейс

Rioskill committed:
Update Wild_Spirits.md

Corrected misspells, added positive cases for event creation and place creation

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -93,7 +93,7 @@
   - Если сайт имеет неверный формат, то после сохранения
     - отображается как до изменения
-    - никаких подсказок не появляется
+    - ${color{red} никаких \ подсказок \ не \ появляется}
```
Латекс не отрендерился, плюс у поля "Телефон" такая же бага, надо выделить

Rioskill replied:
Спасибо, исправил

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -141,10 +141,22 @@
+  - Поле "Дата начала" содержит время
+  - Поле "Категория" содержит одно из следующих значений: "Концерты", "Театр", "Кино", "Фестивали", "Выставки"
+  - Место проведения выбрано
+  - Картинка выбрана
```
Забыл 2 пробела для переноса строки

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -141,10 +141,22 @@
 ## Создание мероприятия
+- При заполнении на странице создания мероприятия полей следующим образом:
+  - Поле "Название мероприятия" содержит названия не используемое уже существующими на сайте мероприятиями
+  - Поле "Дата начала" содержит дату в будущем
+  - Поле "Дата начала" содержит время
```
Имеешь в виду "Время начала"?

reddiridabl666 commented on Wild_Spirits.md:
```diff
@@ -141,10 +141,22 @@
+- При заполнении на странице создания мероприятия полей следующим образом:
+  - Поле "Название мероприятия" содержит названия не используемое уже существующими на сайте мероприятиями
+  - Поле "Дата начала" содержит дату в будущем
+  - Поле "Дата начала" содержит время
```
Имеешь в виду "Время начала"?

Rioskill replied:
Ага, изменил

Rioskill committed:
Update Wild_Spirits.md

Поправил вёрстку

CyTb-On-Fire commented on Wild_Spirits.md:
```diff
@@ -84,6 +84,18 @@
   - ${\color{red} Нет \ стилей \ для \ селекта }$
   - ${\color{red} После \ сохранения \ ни \ на \ что \ не \ влияет }$
 - При клике на "Отменить" все изменения отменяются
+- Телефон
```
Нет проверки с разной длиной вводимого значения

CyTb-On-Fire commented on Wild_Spirits.md:
```diff
@@ -84,6 +84,18 @@
+    - ${/color {red} никаких \ подсказок \ не \ появляется}$
+  - Если телефон имеет верный формат, то после сохранения
+    - ${\color {red} Поле  \ "Телефон" \ пропадает \ и \ появляется \ только \ после \ обновления \ страницы}$
+- Сайт
```
Как и у телефона нет проверки с разными длинами.

CyTb-On-Fire commented on Wild_Spirits.md:
```diff
@@ -123,4 +135,106 @@
+  - Поле "Время начала" содержит время
+  - Поле "Категория" содержит одно из следующих значений: "Концерты", "Театр", "Кино", "Фестивали", "Выставки"
+  - Место проведения выбрано
+  - Картинка выбрана
```
Нет теста при загрузке картинки слишком большого размера

Rioskill committed:
Добавил проверки на длины полей

Rioskill committed:
Добавил кейс с пустым полем тегов

Rioskill committed:
Устранил ошибки в латексе

reddiridabl666 reviewed (approved):
Апрув

mihaillatyshov committed:
Merge branch 'Wild_Spirits' into Wild_Spirits
